public/app/core/utils/kbn.js

```diff
@@ -714,12 +719,17 @@ function($, _) {
       {
         text: 'currency',
         submenu: [
-          {text: 'Dollars ($)', value: 'currencyUSD'},
-          {text: 'Pounds (£)',  value: 'currencyGBP'},
-          {text: 'Euro (€)',    value: 'currencyEUR'},
-          {text: 'Yen (¥)',     value: 'currencyJPY'},
-          {text: 'Rubles (₽)',  value: 'currencyRUB'},
-          {text: 'Hryvnias (₴)',  value: 'currencyUAH'},
+          {text: 'Dollars ($)',        value: 'currencyUSD'},
+          {text: 'Pounds (£)',         value: 'currencyGBP'},
+          {text: 'Euro (€)',           value: 'currencyEUR'},
+          {text: 'Yen (¥)',            value: 'currencyJPY'},
+          {text: 'Rubles (₽)',         value: 'currencyRUB'},
+          {text: 'Hryvnias (₴)',       value: 'currencyUAH'},
+          {text: 'Polish złoty (zł)',  value: 'currencyPLN'},
+          {text: 'Indian rupee (₹)',   value: 'currencyINR'},
+          {text: 'Israeli shekel (₪)', value: 'currencyILS'},
+          {text: 'Turkish lira (₺)',   value: 'currencyTRY'},
+          {text: 'Bitcoin (₿)',        value: 'currencyBTC'},
```

Comment:
What about using B͈̎ here?

Reply:
I've used the Unicode symbol for Bitcoin. Using old capital "b" with strokes isn't necessary.